src/ponyc/CMakeLists.txt

```diff
@@ -26,24 +28,16 @@ if (MSVC)
     )
 
     set_source_files_properties(main.c PROPERTIES LANGUAGE CXX)
-
-    llvm_config(ponyc all)
-
-    get_target_property(TARGET_LIBRARIES ponyc LINK_LIBRARIES)
-    LIST(REMOVE_ITEM TARGET_LIBRARIES LTO)
-    set_property(TARGET ponyc PROPERTY LINK_LIBRARIES ${TARGET_LIBRARIES})
 else()
     target_link_libraries(ponyc PRIVATE
         libponyc
         libponyrt
         blake2
     )
 
-    llvm_config(ponyc all)
-
     if(${CMAKE_HOST_SYSTEM_NAME} MATCHES "Darwin")
-        target_link_libraries(ponyc PRIVATE dl)
-    elseif(${CMAKE_HOST_SYSTEM_NAME} MATCHES "BSD")
+        target_link_libraries(ponyc PRIVATE dl z)
+    elseif(${CMAKE_HOST_SYSTEM_NAME} MATCHES "BSD|DragonFly")
```

Comment:
the next line is for DragonFly. I don't believe this change is correct.

Reply:
Another merge issue; will fix.